src/coreclr/jit/simdcodegenxarch.cpp

```diff
@@ -70,6 +47,13 @@ void CodeGen::genStoreIndTypeSimd12(GenTreeStoreInd* treeNode)
     GenTree* addr = treeNode->Addr();
     genConsumeAddress(addr);
 
+    if (addr->isContained() && addr->OperIs(GT_LCL_ADDR))
+    {
+        genEmitStoreLclTypeSimd12(treeNode, addr->AsLclFld()->GetLclNum(), addr->AsLclFld()->GetLclOffs());
+        genUpdateLife(treeNode);
+        return;
+    }
```

Comment:
An alternative fix to this would have been to simply disallow containment of local addresses for `SIMD12`s.

This would work because accurate liveness tracking for `SIMD12`s is neither used not important, but it felt more consistent with the rest of codegen to handle it. Perhaps, one day, we'll delete this code.

Reply:
It's quite confusing that `emitHandleMemOp` does not support contained `LCL_ADDR` nodes when `Lowering::ContainCheckIndir` does. I wonder if we have more bugs around this.